project/plugins.sbt

```diff
@@ -6,35 +6,24 @@ resolvers += "sonatype-releases" at "https://oss.sonatype.org/content/repositori
 
 resolvers += "Spark Package Main Repo" at "https://dl.bintray.com/spark-packages/maven"
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.12.0")
+libraryDependencies += "org.ow2.asm"  % "asm" % "5.0.3"
+
+libraryDependencies += "org.ow2.asm"  % "asm-commons" % "5.0.3"
 
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "4.0.0")
 
 addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.6.0")
 
-// For Sonatype publishing
-//resolvers += Resolver.url("sbt-plugin-releases", new URL("http://scalasbt.artifactoryonline.com/scalasbt/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
 
-//addSbtPlugin("com.jsuereth" % "xsbt-gpg-plugin" % "0.6")
+addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.2")
```

Comment:
version 0.2.3 is out now, I think, but I don't think it has any changes relevant to this library (only R stuff).